### DISCUSSION TITLE
Outline not displaying document bodies made by other users

### DISCUSSION THREAD

**Discussion by burritosoftware** [outline/outline#6395] [Self Hosting] [answered]
Hi everyone, I just setup Outline with Docker and Cloudflare Tunnel alongside SSO for a small team. When I create documents with my Admin account, others can see the body of the document. But when others create documents, none of us, including me, can see the body of the document: only the title.

I have Rocket Loader disabled, and I also tried checking if it was a tunnel issue by just pointing directly to my VPS through DNS and then enabling the https-portal container in the compose file. Both have resulted in only the creator of the documents only being able to see the bodies, but not anyone else; others only see the title.

Kinda stuck here, any advice?

**Comment by tommoor** [comment]
Sounds like the websocket connection is being blocked to me, are you not seeing failed requests in browser developer tools?

**Reply by burritosoftware** [reply, marked as answer]
I was seeing errors before about Firefox upgrading requests to be wss instead of ws. I originally made a single configuration rule to disable Cloudflare’s Rocket Loader just for the site, but then I disabled it globally, and ran outline through Cloudflare Tunnel instead of https-portal. Somewhere, somehow, it was working, without any WebSockets errors.

I’m not sure where the fault laid as Cloudflare had websockets turned on the entire time and the issue appeared for both standard setup with https-portal and with CF Tunnel. I guess I can just blame Rocket Loader cause I’m not entirely sure 😅
